### PR TITLE
create Index controller

### PR DESCRIPTION
**Pull request by baghaee-sr**

### CODE REVIEW

File: src/Assets.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public $sourcePath = '@sadi01/bidashboard/assets/bidashboard/';
+
+    public $css = [
+    ];
+    public $js = [
+    ];
+    public $depends = [
+        'yii\web\YiiAsset',
+        'yii\bootstrap5\BootstrapAsset',
```
Comment (Sadi01) [inline src/Assets.php]: Bootstrap4

File: src/BiAssets.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        'bidashboard/dist/js/app.init.mini-sidebar.js',
+        'bidashboard/libs/perfect-scrollbar/dist/perfect-scrollbar.jquery.min.js',
+        'bidashboard/dist/js/sidebarmenu.js',
+        'bidashboard/dist/js/custom.min.js',
+        'bidashboard/libs/fullcalendar/dist/fullcalendar.min.js',
+
+    ];
+    public $depends = [
+        'yii\web\YiiAsset',
+//        'yii\bootstrap5\BootstrapAsset',
```
Comment (Sadi01) [inline src/BiAssets.php]: Remove all unused commented codes

File: src/Assets.php (hunk 1 above)
Comment (baghaee-sr) [reply src/Assets.php]: oky, fixed in new commit

File: src/BiAssets.php (hunk 2 above)
Comment (baghaee-sr) [reply src/BiAssets.php]: oky, removed in new commit

File: src/views/layouts/bid_main.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
+<?php
+
+use Yii;
+use yii\bootstrap5\Html;
```
Comment (Sadi01) [inline src/views/layouts/bid_main.php]: Bootstrap 4

**Review by Sadi01 — COMMENTED** [review]
plz remove all unused assests such as images, icons, css files, js files and ...

File: src/views/layouts/bid_main.php (hunk 3 above)
Comment (baghaee-sr) [reply src/views/layouts/bid_main.php]: fixed

**Review by Sadi01 — APPROVED** [review]

**Pull request merged**